code/headers/beetle.hpp

```diff
@@ -55,7 +55,8 @@ namespace r2d2::moving_platform {
         void move(int8_t distance) override;
         /**
          * @brief
-         * This function will move the platform following a forwards and a backwards
+         * This function will move the platform following a forwards and a
+         * backwards
```

Comment:
A bit nitpicky:  What does this function do exactly?
The sentence is a bit confusing. I assume that this function will move the platform forwards or backwards.